.github/workflows/ci_cd.yml

```diff
@@ -496,6 +496,31 @@ jobs:
           apt install -y tini
           npm install -g @mermaid-js/mermaid-cli
 
+      - name: Install gh cli
+        run: |
+          (type -p wget >/dev/null || (apt update && apt-get install wget -y)) \
+          && mkdir -p -m 755 /etc/apt/keyrings \
+          && wget -qO- https://cli.github.com/packages/githubcli-archive-keyring.gpg | tee /etc/apt/keyrings/githubcli-archive-keyring.gpg > /dev/null \
+          && chmod go+r /etc/apt/keyrings/githubcli-archive-keyring.gpg \
+          && echo "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/githubcli-archive-keyring.gpg] https://cli.github.com/packages stable main" | tee /etc/apt/sources.list.d/github-cli.list > /dev/null \
+          && apt update \
+          && apt install gh -y
+
+      - name: Install quarto to build cheatsheet
+        run: |
+          echo ${{ secrets.GITHUB_TOKEN }} | gh auth login --with-token
+          gh release download --repo github.com/quarto-dev/quarto-cli --pattern *linux-amd64.deb
```

Comment:
Wow! I didn't know gh-cli supported downloads from repos. Cool!

Reply:
https://github.com/quarto-dev/quarto-actions/blob/2b90fd62e27d260d01f63fff6364baa7c2f8d96f/setup/action.yml#L47, yes, it is what quarto setup action using. for some reason, we were not able to add that action here.